### ISSUE TITLE
The newbie: sudo chown -Rh <username> .meteor/local!

### ISSUE THREAD

**Issue by thearabbit** [disney/meteor-base#14]
I tried in your example, but get this
```js
Even with METEOR_ALLOW_SUPERUSER or --allow-superuser, permissions in your app
directory will be incorrect if you ever attempt to perform any Meteor tasks as
a normal user. If you need to fix your permissions, run the following command
from the root of your project:

  sudo chown -Rh <username> .meteor/local
```
Please help me

**Comment by banjerluke** [comment]
I get this message too. I assume it's just a warning message that could theoretically be ignored, since no other Meteor commands are run after the build step anyway.

However, the problem I have is that _nothing happens after this message is printed_, and after a minute or two it is "Killed" by Docker. 🙁

**Comment by banjerluke** [comment]
I figured out the problem. The Docker VM on my computer was only configured to have 1 GB of RAM, which apparently isn't enough to build my app; `meteor build` ran out of memory and silently failed.

Solution was to increase the amount of memory allocated in my Docker VM. In my case, this was through VirtualBox.

(Additionally, it then started running out of disk space, although at least I got a helpful error message about that. Solution was to run `docker system prune` to clear out some space in the Docker VM.)
